server/docs/getting-started.md

```diff
@@ -2,106 +2,251 @@
 
-To streamline the guide, we’ve built a bot to automatically accept requests to play High Roller; this means you only have to code the UI for the first player (the one who proposes to play the game).
+## How does Counterfactual work?
 
-### How does Counterfactual work?
+### For Users
 
-For users, Counterfactual provides a safe central wallet in MetaMask for state channel funds and for tracking state in open channels.
+Users interact with Counterfactual dapps through their wallet-enabled web browser. In this Getting Started Guide, we'll be using  MetaMask and the Counterfactual MetaMask plugin. Check out the installation guide [here](https://github.com/counterfactual/monorepo/tree/master/packages/cf-metamask-extension).
 
-For developers, Counterfactual disentangles the UI of the dapp from the formal logic of the game being implemented.
+INSERT DIAGRAM-USER HERE
 
+Counterfactual is designed build web apps with a [hub-and-spoke model](https://medium.com/blockchannel/state-channels-for-dummies-part-3-10b25f6c08b). For us just means that each user already has an established ledger channel with a server-like entity we call the Intermediary. Each user's browser wallet will contain a dedicated ethereum address that contains available funds for dapps hosted by a given Intermediary. In this Getting Started Guide, we'll use the Counterfactual Playground Server as our Intermediary; you can sign up for an account [here](https://playground.counterfactual.com/).
 
-### For Users
+When two users agree to install an instance of a dapp via the Playground, the Playground will allocate funds from their dedicated addresses into a virtual channel for the dapp instance.
+
+INSERT DIAGRAM FUNDVCHANN HERE
 
-Users install the Counterfactual MetaMask plugin, which creates an Ethereum account in MetaMask managed by the Counterfactual Node. Funds in this account are used to fund the individual channels that players choose to open in any specific Counterfactual app.
 
 ### For Developers
 
-The backend logic for the game is implemented by pure functions in the Solidity contract, **HighRoller.sol**. Functionality handled includes answering the following questions:
+
+Developers will use the Counterfactual framework to implement the logic of virtual state channels. In this Getting Started Guide, you'll learn how to 
+
+1. designate the **application logic** for the virtual channel
+1. design the **user interface** for the virtual channel
+
+The application logic for a Counterfactual dapp is described in a deployed Ethereum contract. Functionality handled includes answering the following questions:
 
 1. Whose turn is it now?
-2. What are the possible actions / moves a user can legally execute?
+2. What are the possible actions a user can legally execute?
 3. When is the game over?
 4. What happens to the stake when the game is over?
```

Comment:
The ordering of these four actions could be improved I think.

1. What is the state machine that describes possible actions a user can execute?
2. What defines whose turn is it at some particular state?
3. What defines a "terminal" or "finalize-able" state?
4. What happens to the staked funds after the state machine is finalized?